packages/components/src/components/teaser-media/teaser-media.stories.ts

```diff
@@ -6,254 +6,237 @@ const { argTypes, parameters } = storybookDefaults('sd-teaser-media');
+      <h3 slot="headline">Headline Media Teaser</h3>
+      <div slot="expandable" class="slot slot--border slot--text h-12">Expandable slot</div>
+    </sd-teaser-media>
+  `
 };
 
-export const Samples = {
-  name: 'Samples: Teaser-Media',
-  parameters: {
-    controls: {
-      disable: true
-    },
-    backgrounds: {
-      default: 'white'
-    }
-  },
-  render: () => {
-    return html`
-      <style>
-        #teaserWithContentPlaceholder::part(media) {
-          flex-grow: 1;
-        }
-      </style>
-      <div class="flex flex-wrap gap-8">
-        <sd-teaser-media variant="gradient-dark" class="min-w-[435px] max-w-4xl">
-          <div slot="media" class="relative">
-            <img
-              class="aspect-video object-cover"
-              src="./placeholders/images/architecture.jpg"
-              alt="A skyline of a city"
-            />
-          </div>
-          <div slot="meta" class="meta-info">
-            <span class="meta-info-item">01.12.2013</span>
-            <span class="meta-info-item">| Author name</span>
-          </div>
-          <h3 slot="headline">Not expandable teaser-media</h3>
-          <div class="flex flex-col gap-4">
-            <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit.</p>
-            <div class="flex-none">
-              <sd-button href="#" target="_blank" variant="primary" inverted>Link</sd-button>
-            </div>
-          </div>
-        </sd-teaser-media>
-        <sd-teaser-media variant="gradient-white" class="min-w-[435px] max-w-4xl">
-          <div slot="media" class="relative">
-            <img
-              class="aspect-video object-cover"
-              src="./placeholders/images/architecture.jpg"
-              alt="A skyline of a city"
-            />
-          </div>
-          <div slot="meta" class="meta-info">
-            <span class="meta-info-item">01.12.2013</span>
-            <span class="meta-info-item">| Author name</span>
-          </div>
-          <h3 slot="headline">Expandable teaser-media</h3>
-          <div slot="expandable">
-            <p>
-              Expandable: Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut
-              labore et dolore magna aliqua.
-            </p>
-          </div>
-          <div class="flex flex-col gap-4">
-            <p>
-              Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et
-              dolore magna aliqua.
-            </p>
-            <div class="flex-none">
-              <sd-button href="#" target="_blank" variant="primary">Link</sd-button>
-            </div>
-            <div class="flex-none text-sm">
-              <p>@Copyright Lorem ipsum</p>
-            </div>
-          </div>
-        </sd-teaser-media>
-      </div>
-    `;
-  }
+/**
+ * `Copyright` can be shown at the bottom of the default slot.
+ */
+
+export const Copyright = {
+  name: 'Copyright (Optional)',
```

Comment:
name: 'Copyright',
  
  I would remove the (Optional) since it says "can be ..."

Reply:
Updated